src/platform/ESP32/nimble/BLEManagerImpl.cpp

```diff
@@ -1057,6 +1055,21 @@ CHIP_ERROR BLEManagerImpl::StartAdvertising(void)
     }
 
 exit:
+    if (err == BLE_HS_EALREADY && connectable)
+    {
+        ChipLogProgress(DeviceLayer,
+                        "Connectable advertising failed because device was already advertising , stop active advertisement");
+        err = ble_gap_adv_stop();
+        if (err != CHIP_NO_ERROR)
+        {
+            ChipLogError(DeviceLayer, "ble_gap_adv_stop() failed: %s", ErrorStr(err));
```

Comment:
@prasad-alatkar  Do you want to continue to start advertisement even if stopping the advertisement has failed?

Reply:
Will make appropriate change. Thanks!!

Reply:
Done!!